wrappers/swift/SDWLogManager.swift

```diff
@@ -0,0 +1,182 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Wrapper over ODWLogManager which manages the telemetry logging system.
+public class LogManager {
+
+    /**
+    Initializes the telemetry logging system with the speicified tenant token and custom configuration.
+
+    - Parameters:
+        - tenantToken: A `String` that contains the tenant token.
+        - config: A `Dictionary` that contains a custom configuration.
+
+    - Returns:
+        - A `Logger` instance.
+    */
+    public static func initForTenant(tenantToken: String!, config: [AnyHashable: Any]? = nil) -> Logger? {
+        if let odwLogger = ODWLogManager.initForTenant(tenantToken, withConfig: config) {
+            return Logger(logger: odwLogger)
+        } else {
+            return nil
+        }
+    }
+
+    /**
+    Retrieves a new instance of `Logger` for logging telemetry events. Initializes the telemetry logging system,
+    if not already initialized, with the default configuration using the specified tenant token and source.
+
+    - Parameters:
+        - tenantToken: A `String` that contains the tenant token.
+        - withSource: A `String` that contains the name of the source of events.
```

Comment:
I'm not quite sure if "with..." applies as well to Swift, like it does with Obj-C, please check Swift parameter naming convention.

Reply:
I did found couple of similar names at https://www.swift.org/documentation/api-design-guidelines/. 
so continued using it.
Mostly it emphasized on having good readability while api call.